holoviews/plotting/widgets/__init__.py

```diff
@@ -194,7 +194,9 @@ def _plot_figure(self, idx):
 
 
     def update(self, key):
-        return self._plot_figure(key)
+        self.plot.update(key)
+        self.plot.push()
+        return 'Complete'
```

Comment:
``'Update Complete'`` maybe? Is 'Complete' by itself informative enough? I'm assuming this might be something that appears in the console. If it is a message/part of a protocol, it is probably not worth changing.